### PR TITLE
Add ut for node.go

### PR DESCRIPTION
**Pull request by jwcesign**
Signed-off-by: jwcesign <jiangwei115@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Part of #2579

**Special notes for your reviewer**:
none

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @XiShanYongYe-Chang

File: pkg/karmadactl/cmdinit/kubernetes/node_test.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+			}
+		})
+	}
+}
+
+func TestCommandInitOption_AddNodeSelectorLabels(t *testing.T) {
+	tests := []struct {
+		name     string
+		option   CommandInitOption
+		nodeName string
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/kubernetes/node_test.go:161]: This field seems can be removed.

**Comment by jwcesign** [comment]
cc @XiShanYongYe-Chang for checking again

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm
The original code logic has some modification, I think it is reasonable, ask for @prodanlabs to take a review.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**